contrib/confluentinc/confluent-kafka-go/kafka/kafka_test.go

```diff
@@ -180,3 +181,76 @@ func TestConsumerPoll(t *testing.T) {
 	assert.Equal(t, "queue", s1.Tag(ext.SpanType))
 	assert.Equal(t, int32(0), s1.Tag("partition"))
 }
+
+func TestConsumerReadMessage(t *testing.T) {
```

Comment:
This test is too identical to the previous one. Please try and find a way to de-dupe it. Perhaps by creating sub-tests and reusing some parts.

Reply:
Refactored, thanks.